src/mlpack/core/util/timers.hpp

```diff
@@ -98,7 +99,7 @@ class Timers
 {
  public:
   //! Default to disabled.
-  Timers() : enabled(false) { }
+  Timers() : enabled(false) {  }
```

Comment:
```suggestion
  Timers() : enabled(false) { }
```
I don't think there's a need for an extra space here. :+1: